tools/clang/test/CodeGenSPIRV/spirv.debug.opline.intrinsic.control.barrier.hlsl

```diff
@@ -0,0 +1,24 @@
+// Run: %dxc -T cs_6_0 -E main -Zi
```

Comment:
This test was not added in `CodeGenSpirvTest.cpp`

Reply:
It was mistake. I just added it.